web/src/components/task/task-sidebar/finish-button/utils.ts

```diff
@@ -1,7 +1,29 @@
+const LISTED_PAGES_LIMIT = 10;
+const SLICE_START_INDEX = 0;
+const SLICE_END_INDEX = LISTED_PAGES_LIMIT - 1;
+
+const getPagesList = (pages: number[]) => pages?.join(', ');
+
+const getAboveLimitPagesCount = (pages: number[]) => pages.length - LISTED_PAGES_LIMIT;
+
+const getVisiblePagesString = (pages: number[]) => getPagesList(getVisiblePagesArray(pages));
+
+const getVisiblePagesArray = (pages: number[]) => pages.slice(SLICE_START_INDEX, SLICE_END_INDEX);
```

Comment:
Optional: I think we can just directly use numbers (based on LISTED_PAGES_LIMIT) instead of "SLICE_START_INDEX" and "SLICE_END_INDEX". This will not look like magic numbers

Reply:
Addressed